test/karma.debug.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" id="html">
+<head>
+%X_UA_COMPATIBLE%
```

Comment:
It's there by default in Karma's debugContextFile, which this file is a derivative of. I don't why don't have it in their non-debug file by default.